src/node_lttng_provider.h

```diff
@@ -78,7 +78,7 @@ void NODE_GC_START(v8::GCType type,
 void NODE_GC_DONE(v8::GCType type,
                   v8::GCCallbackFlags flags,
                   v8::Isolate* isolate) {
-  const char* typeStr = "Unkown GC Type";
+  const char* typeStr = "Unknown GC Type";
```

Comment:
Well it's a tracing mechanism, which may/may not be considered under part of the "runtime." That's why I'd like to get some feedback to see where this kind of change would fall since AFAIK error string changes in general are still semver-major changes.